judo-runtime-core-jsl-itest/models/SpecialCasesModel/src/test/resources/SpecialCases.jsl

```diff
@@ -122,4 +122,44 @@ entity Case {
 }
 
 entity Static {
-}
\ No newline at end of file
+}
+
+entity PrimUpper {
+    field String Abc;
```

Comment:
If supported by JSL, please add another ref/field, etc with the name of "abc". If we are case sensitive, we need to make sure that different fields with different casings can have dedicated values.

Reply:
Not supported: using the same field name when the only difference is the capitalization.

Reply:
![Screenshot 2024-03-12 at 12 59 37](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/assets/114913193/15a5bd39-4bfb-41d7-b218-169bdeea2d04)